src/options.cpp

```diff
@@ -1470,6 +1476,14 @@ void register_options(void)
                   "Control whether to indent the code between #if, #else and #endif.");
    unc_add_option("pp_define_at_level", UO_pp_define_at_level, AT_BOOL,
                   "Whether to indent '#define' at the brace level (true) or from column 1 (false)");
+   unc_add_option("pp_indent_case", UO_pp_indent_case, AT_BOOL,
+                  "Overrides the indent from pp_if_indent_code for case statements");
+   unc_add_option("pp_indent_func_def", UO_pp_indent_func_def, AT_BOOL,
+                  "Overrides the indent from pp_if_indent_code for function definitions");
+   unc_add_option("pp_indent_extern", UO_pp_indent_extern, AT_BOOL,
+                  "Overrides the indent from pp_if_indent_code for extern C blocks");
+   unc_add_option("pp_indent_brace", UO_pp_indent_brace, AT_BOOL,
+                  "Overrides the indent from pp_if_indent_code for braces");
```

Comment:
the new options should state their default option in the description. I know it's not great but it's what we have.